health/micro-ui/web/core-ui/install-deps.sh

```
@@ -0,0 +1,18 @@
+
+INTERNALS="micro-ui-internals"
+cd ..
+
+cp core/App.js src
+cp core/package.json package.json 
+cp core/webpack.config.js webpack.config.js 
+cp core/inter-package.json $INTERNALS/package.json
+
+cp $INTERNALS/example/src/UICustomizations.js src/Customisations
+
+echo "UI :: core " && echo "Branch: $(git branch --show-current)" && echo "$(git log -1 --pretty=%B)" && echo "installing packages" 
```

Shellcheck (0.10.0)

[style] Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.

(SC2005)